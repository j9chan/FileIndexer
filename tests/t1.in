Ad hoc testing and exploratory testing are important methodologies for checking software integrity, because they require less preparation time to implement, while the important bugs can be found quickly. In ad hoc testing, where testing takes place in an improvised, impromptu way, the ability of a test tool to visually record everything that occurs on a system becomes very important in order to document the steps taken to uncover the bug.