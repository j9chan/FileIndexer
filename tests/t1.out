4 testing
4 the
4 to
3 important
3 in
2 a
2 ad
2 hoc
1 ability
1 an
